bin/eslint.js

```diff
@@ -168,12 +145,9 @@ runWithTiming("engineConfig", function () {
       buildFileList = inclusionBasedFileListBuilder(
         engineConfig.include_paths
       );
-    } else if (engineConfig.exclude_paths) {
-      var ignores = engineConfig.exclude_paths;
-      buildFileList = exclusionBasedFileListBuilder(ignores);
     } else {
-      // No includes or excludes, let's try with everything
-      buildFileList = exclusionBasedFileListBuilder([]);
+      // No explicit includes, let's try with everything
+      buildFileList = inclusionBasedFileListBuilder(["./"]);
```

Comment:
This has been the default behavior in many of our engines, including this one, for a while: there are no production instances where it should occur, but it continues to seem like a reasonable default behavior in the absence of anything explicit.